src/main/java/Panna.java

```diff
@@ -0,0 +1,161 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+// to add: tag for A-Classes
+// to add: tag for A-Collections
+public class Panna {
+
+    private static String command = "";
+
+
+
+    public static void main(String[] args) throws PannaException {
+
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        System.out.println("----------------------------------------------------------\n"
+        + "Hello! I'm Panna.\n" +
+                "What can I do for you?\n\n" +
+                "----------------------------------------------------------");
+
+
+        Scanner s = new Scanner(System.in);
+
+        Panna.command = s.nextLine();
+
+        while (!Panna.command.equals("bye")) {
+
+
+
+
+
+            if (Panna.command.equals("list")) {
+                System.out.println("----------------------------------------------------------");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println(i+1 + ". " + tasks.get(i));
+                }
+                System.out.println("----------------------------------------------------------");
+            }
+
+
+            else if (Panna.command.equals("mark")) {
+                System.out.println("Which one should I mark? Write the label number :] ");
+                try {
+                    int label = s.nextInt();
+                    tasks.get(label - 1).setDone(true);
+
+                    System.out.println("----------------------------------------------------------");
+                    System.out.println("Nice! I've marked this task as done: \n"
+                            + tasks.get(label - 1));
+                    System.out.println("----------------------------------------------------------");
+
+                }
+                catch (Exception e) {
+                    throw new PannaException("Invalid label! The number of tasks now is " + tasks.size() +
```

Comment:
I like how a new PannaException is created to encapsulate errors thrown while running the Panna program.